tools/refguide_check.py

```diff
@@ -15,12 +17,13 @@
 another function, or deprecated, or ...)
 
 Another use of this helper script is to check validity of code samples
-in docstrings. This is different from doctesting [we do not aim to have
-numpy docstrings doctestable!], this is just to make sure that code in
-docstrings is valid python::
+in docstrings::
 
-    $ python refguide_check.py --doctests optimize
+    $ python refguide_check.py --doctests ma
 
+or in RST-based documentations (including any python files in the directories)::
```

Comment:
We removed the python file handling, right?

Reply:
fixing